docs/integrations/python.md

```diff
@@ -11,11 +11,12 @@ redirect_from:
   - /integrations/boto.html
 ---
 
-{% include toc.html %}
+# Use Python to interact with your objects on lakeFS
 
-## Boto vs. lakeFS SDK
+{% include toc_2-3.html %}
+
+There are two primary ways to work with lakeFS from Python: 
```

Comment:
I feel boto should be a different integration, we should probably default to the native python SDK. (again we can discuss outside this PR)

Reply:
Tracking: https://github.com/treeverse/lakeFS/issues/6056